insights_messaging/appbuilder.py

```diff
@@ -133,8 +135,17 @@ def build_app(self):
         if log_config:
             logging.config.dictConfig(log_config)
         else:
+            handler = logging.StreamHandler(stream=sys.stdout)
+            handler.setFormatter(LogstashFormatterV1()) # to keep the same format with cloud env
             logging.basicConfig(level=logging.DEBUG)
-
+            logging.getLogger("insights.core.dr").setLevel(logging.ERROR)
```

Comment:
Add a comment to explain what the handler at index 0 is and why removing it is necessary.